esp32-box-docks/box3-sensor-generic.yaml

```
@@ -0,0 +1,116 @@
+# This firmware is soley for the Espressif ESP32-S3-Box-3 with the Sensor dock
+# It sets up sensors and configuration entities in Home Assistant for this specific hardware
+
+substitutions:
+  name: esp32-s3-box-3-sensor
+  friendly_name: ESP32 S3 Box 3 Sensor
+
+esphome:
+  name: "${name}"
+  friendly_name: "${friendly_name}"
+  name_add_mac_suffix: true
+  platformio_options:
+    board_build.flash_mode: dio
+  project:
+    name: esphome.esp_web_tools_example
+    version: "1.0"
+
+esp32:
+  board: esp32-s3-devkitc-1
+  framework:
+    type: arduino
+
+# Enable logging
+logger:
+
+# Enable Home Assistant API
+api:
+
+# Allow OTA updates
+ota:
+
+# Allow provisioning Wi-Fi via serial
+improv_serial:
+
+wifi:
+  # Set up a wifi access point
+  ap:
+    ssid: "ESP Web Tools"
+
+# In combination with the `ap` this allows the user
+# to provision wifi credentials to the device via WiFi AP.
+captive_portal:
+
+dashboard_import:
+  package_import_url: github://esphome/firmware/esp-web-tools/esp32s3.yaml@v2
```

This needs to match the path of this file
```suggestion
  package_import_url: github://esphome/firmware/esp32-box-docks/box3-ssensor-generic.yaml@main
```

---

typo corrected on `ssensor`